inc/TRestAxionLikelihood.h

```diff
@@ -81,7 +81,7 @@ class TRestAxionLikelihood : public TRestMetadata {
 
     Double_t GetSignal(Double_t ma, Double_t g10_4, Double_t rho, Double_t tExp);
 
-    void LikelihoodTest(string fname);
+    void LikelihoodTest(std::string fname);
```

Comment:
Let try to always use constant references in functions instead of copying the value.

This should be:

```
void LikelihoodTest(const std::string& fname);
```

In this case the performance gain is minimal but it helps give a consistent style in the framework.